goconvey.go

```diff
@@ -269,10 +271,19 @@ func getWorkDir() string {
 	return working
 }
 
+func parseTags() []string {
+	gotags = strings.TrimSpace(gotags)
+	if len(gotags) == 0 {
+		return make([]string, 0, 0)
+	}
+	return strings.Split(gotags, ",")
```

Comment:
This is likely a result of my brain frying with the conversion between comma- and space-separated. I will update.